bfsfs/bucket.go

```diff
@@ -96,8 +97,3 @@ func (b *bucket) Remove(ctx context.Context, name string) error {
 func (b *bucket) Close() error {
 	return nil // noop
 }
-
-// resolve returns full safely rooted path.
-func (b *bucket) resolve(name string) string {
-	return filepath.Join(b.root, filepath.Join("/", name))
```

Comment:
`path/filepath` makes sense to deal with filesystem, actually. Or you dropped non-UNIX-FS support on purpose?

Reply:
updated